guides/common/modules/proc_upgrading-the-external-database.adoc

```diff
@@ -1,25 +1,22 @@
 [id="Upgrading_the_External_Database_{context}"]
 = Upgrading the external database
 
-You can upgrade an external database from {EL} 8 to {EL} 9 while upgrading {Project} from {ProjectVersionPrevious} to {ProjectVersion}.
+If your {Project} deployment uses an external database, the database will be upgraded when running {foreman-installer} on the {Project} server.
 
 .Prerequisites
-* Create a new {EL} 9 based host for PostgreSQL server that follows the external database on {EL} 9 documentation.
-ifdef::katello,orcharhino,satellite[]
-For more information, see {InstallingServerDocURL}using-external-databases_{project-context}[Using External Databases with {Project}].
-endif::[]
-* Install PostgreSQL version 13 on the new {EL} host.
+* PostgreSQL version 13 is installed on the {EL} host.
 
 .Procedure
-. Create a backup.
-. Restore the backup on the new server.
-. If {Project} reaches the new database server via the old name, no further changes are required.
-Otherwise reconfigure {Project} to use the new name:
+. Update permissions to prepare the *postgresql-evr* extension for removal:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} \
---foreman-db-host _newpostgres.example.com_ \
---katello-candlepin-db-host _newpostgres.example.com_ \
---foreman-proxy-content-pulpcore-postgresql-host _newpostgres.example.com_
+# runuser -l postgres -c \
+"psql -d foreman -c \"UPDATE pg_extension SET extowner = (SELECT oid FROM pg_authid WHERE rolname='foreman') WHERE extname='evr';\""
 ----
+ifdef::upgrading-connected[]
```

Comment:
I think something is still wrong here:
```
<p>Follow <a href="#upgrading_a_disconnected_foreman_server_upgrading-connected">[upgrading_a_disconnected_foreman_server_upgrading-connected]</a>.
make[1]: *** [../common/Makefile:82: ../build/Upgrading_Project/index-foreman-el.html] Error 1
make[1]: Leaving directory '/home/runner/work/foreman-documentation/foreman-documentation/guides/doc-Upgrading_Project'
make: *** [Makefile:13: doc-Upgrading_Project] Error 2
make: *** Waiting for unfinished jobs....
```
It looks like the context isn't being checked correctly.